src/taco-cli/cli/commands.json

```diff
@@ -399,6 +399,9 @@
             ,
             {
                 "name": "--target=TARGET", "description": "[CommandRunTargetDescription]"
+            },
+            {
+                "name":  "--list", "description": "[CommandRunListDescription]"
```

Comment:
Yes, let us atleast be explicit that when not specifying the device/emulator options in conjunction with the --list option, both targets will be listed.